CHANGELOG.md

```diff
@@ -6,7 +6,7 @@
 
 #### Fixes
 
-* Your contribution here.
```

Comment:
Don't remove this line next time ;)

Reply:
🙈

Reply:
Let's say that I believe that there will only be new features in the next release and no fixes 🤞.

Reply:
Sorry about that @dblock

Reply:
It looks like a new "check" can be added to the "danger-changelog" gem.

Reply:
For sure, we take PRs :)